cmd/agent/app/run.go

```diff
@@ -345,9 +346,14 @@ func StartAgent() error {
 		orchestratorForwarder.Start() //nolint:errcheck
 	}
 
+	common.EventPlatformForwarder = epforwarder.NewEventPlatformForwarder()
```

Comment:
Could we return an error too ? And maybe print it ?

Reply:
What kind of error? There is no error that would make `EventPlatformForwarder` cause problems at this level. 

The worst case scenario is if all of its internal pipelines fail to be instantiated because of some configuration issue in which case the `EventPlatformForwarder` would basically be a noop. 

There is logging within `epforwarder.NewEventPlatformForwarder()` that will alert the user if there's a configuration issue with any of the pipelines.